pdata/internal/wrapper_traces.go

```diff
@@ -20,15 +20,24 @@ import (
 )
 
 type Traces struct {
-	orig *otlpcollectortrace.ExportTraceServiceRequest
+	orig  *otlpcollectortrace.ExportTraceServiceRequest
+	state *State
```

Comment:
state State

Reply:
Resolved. I applied the changes that you suggested on the call